wurst/systems/spells/priest/IncreaseMetabolism.wurst

```diff
@@ -46,4 +46,4 @@ init
     onUnitHealed() ->
         let instance = getHealingInstance()
         if instance.target.hasAbility(BUFF_ID)
-            instance.increaseMultiplier(0.5)
+            instance.scaleMultiplier(1.5)
```

Comment:
Does this affect all healing or only item healing?

Reply:
Fixed